platform/device/builtin/db.go

```diff
@@ -46,6 +47,24 @@ func NewDB(db *bolt.DB) (*DB, error) {
 	return datastore, nil
 }
 
+func (db *DB) GetBootstrapToken(ctx context.Context, udid string) ([]byte, error) {
+	opts := device.ListDevicesOption{}
+	
+	var udids []string
+	udids = append(udids, udid)
+
+	opts.FilterUDID = udids
+	
+	devices, err := db.List(ctx, opts)
+	for _, d := range devices {
+		btstring, _ := hex.DecodeString(string(d.BootstrapToken))
```

Comment:
Any reason to store/encode/decode as hex? We can store the raw binary in the device record.